lib/ansible/modules/files/xattr.py

```diff
@@ -177,9 +198,9 @@ def main():
     if key is None and state in ['absent', 'present']:
         module.fail_json(msg="%s needs a key parameter" % state)
 
-    # All xattr must begin in user namespace
-    if key is not None and not re.match(r'^user\.', key):
-        key = 'user.%s' % key
+    # Prepend the key with the namespace if defined
+    if key is not None and namespace is not None and len(namespace) > 0:
```

Comment:
`... and not (namespace == 'user' and re.match(r'^user\.', key)):

Reply:
OK, I will do that. But I will use `key.startswith('user.')` instead of the regexp.

Reply:
works4me